### ISSUE TITLE
[wii] 0.9.3 Time settings?

### ISSUE THREAD

**Issue by almircanella** [mgba-emu/mgba#2422]
Hello! First, this is a great emulator and I have a lot of fun playing on my wii, 240p goodnes.

1)  .sav files are generated with the year 2030 and weird clock time is shown in mgba menu. My wii date settings are just fine.
some workaround? This is RTC ?

2) the Menu is very strange with scale set to 1.0, pixelated. In game ok text Is so out of the screen, but games are ok and centered. My wii is using component cables on 480i CRT. 

3) I've played some games on mgba via retroarch (nintendo switch) and .srm files were generated. How can I use them on mgba wii, it only recognizes .sav files ?

Thanks in advance.

**Comment by endrift** [comment]
1. Unsure what's going on here. Do other versions of mGBA still have this issue?
2. Sorry, I'm having trouble understanding what you're saying, presumably due to the language barrier. Can you rephrase that?
3. Rename the file from .srm to .sav. They have more or less the same contents. There's also mGBA standalone on Switch.
